src/_common/block/form/form.ts

```diff
@@ -1,17 +1,27 @@
 import Component from 'vue-class-component';
 import { Prop } from 'vue-property-decorator';
 import { Api } from '../../api/api.service';
+import AppFormControlToggle from '../../form-vue/control/toggle/toggle.vue';
 import { BaseForm, FormOnSubmit } from '../../form-vue/form.service';
 import { User } from '../../user/user.model';
 
-@Component({})
-export default class AppBlockForm extends BaseForm<any> implements FormOnSubmit {
+type BlockData = {
```

Comment:
make an interface and call it FormModel, I think that's our convention, yeah?

Reply:
We do both interface and type, I'll rename it tho.